templates/clients/show.twig

```diff
@@ -63,9 +63,11 @@
                     <a href="{{ moduleurlpath }}/clients/" class="ui small primary labeled icon button">
                         <i class="arrow left icon"></i> {% trans '{oidc:return}' %}
                     </a>
+                    <a href="{{ moduleurlpath }}/clients/edit.php?client_id={{ client.identifier }}" class="ui small labeled icon button">
```

Comment:
The preferred way to do module urls is:
{{ moduleURL('oidc/clients/edit.php?client_id=' ~ client.identifier|escape('url')) }}

There is a TwigFilter for assets as well:
{{ asset('clipboard/clipboard.min.js', 'oidc') }}
You may even use the Clipboard-JS from SSP instead of including your own copy